cabal.project

```diff
@@ -190,12 +194,13 @@ source-repository-package
 
 source-repository-package
   type: git
-  location: https://github.com/raduom/iohk-monitoring-framework
-  tag: b5c035ad4e226d634242ad5979fa677921181435
+  location: https://github.com/j-mueller/iohk-monitoring-framework
```

Comment:
What's the issue here?

Reply:
Needed to increase the bounds on `base` to build with newer GHC. I can make a PR into @raduom 's fork .

Reply:
It's fine. I don't care much for maintaining it. :)